regolith/helpers/u_contacthelper.py

```diff
@@ -0,0 +1,131 @@
+"""Helper for adding a new person to the contacts collection.
+
+"""
+import datetime as dt
+from nameparser import HumanName
+import dateutil.parser as date_parser
+import sys
+import uuid
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import all_docs_from_collection
+
+
+TARGET_COLL = "contacts"
+
+def subparser(subpi):
+    subpi.add_argument("name", help="first name space last name in quotes")
+    subpi.add_argument("institution", help="person's institution.  Can be inf"
+                                            "short form such as columbiau and will "
+                                            "be retrieved from institutions collection")
+    # subpi.add_argument("-e", "--email",
+    #                    help="email address")
+    subpi.add_argument("-a", "--aliases", nargs='+',
+                        help="All the different ways that the person may "
+                             "be referred to as.  As many as you like, in "
+                             "quotes separated by commas")
+    subpi.add_argument("-n", "--notes", nargs='+',
+                        help="notes.  As many notes as you like, each one in "
+                             "quotes and separated by a comma, such as where"
+                             "and when met, what discussed.")
+    # Do not delete --database arg
+    subpi.add_argument("--database",
+                       help="The database that will be updated.  Defaults to "
+                            "first database in the regolithrc.json file.")
+    # Do not delete --date arg
+    subpi.add_argument("--date",
+                       help="The date when the contact was created in ISO format"
+                            " Defaults to today's date."
+                       )
+    subpi.add_argument("--update",
```

Comment:
ok, Professor! I added them there so I could pass the tests, otherwise it generates a new uuid and updated date and time. Do you have any solution for that?

Reply:
ah, good point.  Let me think about it.

Reply:
here is the answer for the uuid/date.  It is a bit tricky, it requires you to "mock" the uuid generation in the test:
https://stackoverflow.com/questions/44395755/how-to-mock-uuid-generation-in-a-test-case
https://stackoverflow.com/questions/23988853/how-to-mock-set-system-date-in-pytest

Have a quick play around with it, but if it is too difficult to understand, make an issue to fix that in your u_contact helper and we can merge it without that and someone else can take it on.